docs/data/date-pickers/timezone/timezone.md

```diff
@@ -0,0 +1,218 @@
+---
+product: date-pickers
+title: Date and Time pickers - UTC and timezones
```

Comment:
![image](https://github.com/mui/mui-x/assets/3309670/987bae9e-60b8-41a9-bc31-5b0c8d51ccea)

Maybe we can do that in another PR since we must change it on all pages

Reply:
Sure thing, I also noticed that the inconsistency is already there 🙈